### PR TITLE
updated billing events

### PR DESCRIPTION
**Pull request by jerae-duffin**
New updates to Billing events

### CODE REVIEW

**Review by leslielylee — CHANGES_REQUESTED** [review]
One change for the "Billing Cycle Changed" description. This happens when the org owner changes their billing cycle (e.g. monthly to annual).

Can we change the description to something like "Occurs when there is a change in the recurring interval that your organization is charged"?

**Review by leslielylee — APPROVED** [review]
LGTM. Please wait for @ob1dev to finish his review prior to merging, thank you!

File: docker-hub/audit-log.md [hunk 1]
```diff
@@ -81,10 +81,10 @@ Refer to the following section for a list of events and their descriptions:
...
-| Cycle Changed | Billing cycle change events |
-| Downgrade Canceled | Billing plan cancel downgrade events |
-| Seat Removal Canceled | Billing cancel seat removal events |
\ No newline at end of file
+| Plan Upgraded | Occurs when your organization’s billing plan is upgraded to a higher tier plan.|
+| Plan Downgraded | Occurs when your organization’s billing plan is downgraded to a lower tier plan. |
+| Seat Added | Occurs when a seat is added to your organization’s billing plan. |
+| Seat Removed | Occurs when a seat is removed from your organization’s billing plan. |
+| Billing Cycle Changed | Occurs when there is a change in the recurring interval that your organization is charged.|
+| Plan Downgrade Canceled | Occurs when a scheduled downgrade for your organization is canceled.|
```
Comment (ob1dev) [inline docker-hub/audit-log.md:89]:
```suggestion
| Plan Downgrade Canceled | Occurs when a scheduled plan downgrade for your organization is canceled.|
```

**Review by ob1dev — APPROVED** [review]
Looks good to me.

**nitpick:** Do we want to add a short overview for each table? 

For example: `Contains all activities related to an organization's billing.` under https://github.com/docker/docker.github.io/blob/master/docker-hub/audit-log.md#billing-events

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**